utils/staking-miner/src/monitor.rs

```diff
@@ -138,6 +138,21 @@ async fn ensure_no_better_solution<T: EPM::Config, B: BlockT>(
 	}
 }
 
+async fn get_latest_head<T: EPM::Config>(
+	rpc: &SharedRpcClient,
+	mode: &str,
```

Comment:
Should be enum? we can probably reuse the same enum in the CLI flags as well.

Reply:
haha, funny thing I just did that in the other miner repo :P